### DISCUSSION TITLE
Mentioning XKit to staff members?

### DISCUSSION THREAD

**Discussion by dullahandame** [AprilSylph/XKit-Rewritten#443] [Lounge]
Hi, I'm on an email thread with a Tumblr staff member regarding accessibility. Some features of XKit (particularly the ability to entirely remove filtered post content from the dashboard, the ability to see alt text, and the ability to turn off fancy font formatting) are shining examples of what I believe Tumblr should strive for, and thus I considered mentioning the extension to Tumblr. However, are you (the creators) alright with that? If you wish that I don't mention the extension explicitly, I could perhaps simply describe the features that Tumblr should implement, without need for example. Thank you for your time!

**Comment by marcustyphoon** [comment]
I'll let April chime in on this, but Staff is certainly aware of this project; in fact, there are pieces of code in the site that specifically help us implement the features that we have!

**Reply by AprilSylph** [reply]
Staff's awareness of XKit is a ship that's long sailed. One of my sponsors here on GitHub is a Tumblr backend engineer, New XKit and XKit Rewritten have been linked to on staff.tumblr.com and wip.tumblr.com respectively, and a Tumblr frontend engineer started a discussion here just last week: #410
